### PR TITLE
Add audiences to new event page

### PR DESCRIPTION
**Pull request by gestchild**
References #2863

## Who is this for?
Designers/Devs

## What is it doing for them?
Adds the audience to the format tag

### CODE REVIEW

File: common/views/components/BasePage/EventPage.js [hunk 1]
```diff
@@ -93,9 +93,24 @@ const EventPage = ({ event }: Props) => {
   /* https://github.com/facebook/flow/issues/2405 */
   /* $FlowFixMe */
   const FeaturedMedia = event.promo && <UiImage tasl={tasl} {...image} />;
-  const formatTag = event.format ? [{text: event.format.title}] : [];
+  const formatTag = event.format ? [{text: event.format.title + audiencesString(event.audiences)}] : [];
   const interpretationsTags = event.interpretations ? event.interpretations.map(i => ({text: i.interpretationType.title})) : [];
   const TagBar = <Tags tags={formatTag.concat(interpretationsTags)} />;
+  function audiencesString(audiences) {
+    if (audiences.length > 0) {
```
Comment (davidpmccormick) [inline common/views/components/BasePage/EventPage.js]: Do you need this `if/else`? If audiences is an empty array, then wouldn't the `reduce` give you back the `''`' you put in as the initial argument?

File: common/views/components/BasePage/EventPage.js [hunk 2]
```diff
@@ -93,9 +93,24 @@ const EventPage = ({ event }: Props) => {
   /* https://github.com/facebook/flow/issues/2405 */
   /* $FlowFixMe */
   const FeaturedMedia = event.promo && <UiImage tasl={tasl} {...image} />;
-  const formatTag = event.format ? [{text: event.format.title}] : [];
+  const formatTag = event.format ? [{text: event.format.title + audiencesString(event.audiences)}] : [];
   const interpretationsTags = event.interpretations ? event.interpretations.map(i => ({text: i.interpretationType.title})) : [];
   const TagBar = <Tags tags={formatTag.concat(interpretationsTags)} />;
+  function audiencesString(audiences) {
```
Comment (jamesgorrie) [inline common/views/components/BasePage/EventPage.js]: Could this function live outside of the `EventPage`?

File: common/services/prismic/events.js [hunk 3]
```diff
@@ -61,14 +61,19 @@ function parseEventDoc(document: PrismicDocument, scheduleDocs?: PrismicFragment
   const eventbriteIdScheme = identifiers.find(id => id.identifierScheme === 'eventbrite-id');
   const eventbriteId = eventbriteIdScheme && eventbriteIdScheme.value;
 
+  const audiences = document.data.audiences.map(audience => isDocumentLink(audience.audience) ? ({
+    title: asText(audience.audience.data.title),
+    description: asText(audience.audience.data.description)
```
Comment (jamesgorrie) [inline common/services/prismic/events.js]: We don't use the description anywhere, but if we did, it should probably just be in prismic format.

Comment (gestchild) [reply common/services/prismic/events.js]: Yep, have changed that here: https://github.com/wellcometrust/wellcomecollection.org/pull/2921/commits/1b84351be0a603b943e9c796eb4365eee6e9e2cc#diff-db1ea758ff420e1c26768041df9cadf8R74

Comment (jamesgorrie) [reply common/services/prismic/events.js]: 👍

**Pull request merged**